meson.build

```diff
@@ -0,0 +1,78 @@
+project('io', 'd', default_options: ['optimization=3'], version: '2.3.3', license: 'BSL-1.0' )
+common_project_arguments = ['-release'] #, '--d-debug'
+add_project_arguments(common_project_arguments, language: 'd')
+
+# semver
+version = meson.project_version().split('.')
+major_version = version[0].to_int()
+minor_version = version[1].to_int()
+micro_version = version[2].to_int()
+
+# base
+dc = meson.get_compiler('d')
+
+# Source files
+
+sources = [ 'src/std/io/driver/package.d',
+            'src/std/io/driver/sync.d',
+            'src/std/io/exception.d',
+            'src/std/io/file.d',
+            'src/std/io/internal/iovec.d',
+            'src/std/io/internal/string.d',
+            'src/std/io/net/addr.d',
+            'src/std/io/net/dns.d',
+            'src/std/io/net/package.d',
+            'src/std/io/net/socket.d',
+            'src/std/io/net/tcp.d',
+            'src/std/io/net/udp.d',
+            'src/std/io/package.d'
+]
```

Comment:
Is there a way to automate this list as well?

Reply:
never try but should be possible by using os.walk function (`python`)
I will give a try

Reply:
So after some search the answer is yes but is not recommended.

It is considered as a bad practice, references:
- https://stackoverflow.com/questions/34441673/can-you-glob-source-code-with-meson
- https://stackoverflow.com/questions/1027247/is-it-better-to-specify-source-files-with-glob-or-each-file-individually-in-cmak/18538444#18538444

So did you want I create both a .bat and .sh files ?

Reply:
Well, we should do what is expected practice. If it is expected that one must explicitly include sources in the source directory (for whatever reason), then I suppose we should just do that. I do not think we should be executing scripts to fetch file lists. I just found it less ideal to require updating build files when you add/remove source. I remember doing that with certain makefiles *cough druntime* and did not like it.

So leave this as is.